### PR TITLE
Feature/am/web guide for blue dot

### PR DESCRIPTION
**Pull request by ammapspeople**

### CODE REVIEW

File: collections/_web/v3/guides/show_users_position.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+layout: tutorial
+nav_weight: 1250
+published: true
+date: 2020-01-31
+---
+## Overview
+
+In this tutorial, you will learn how to show a dot on the map, representing the user's current location.
+
+The tutorial assumes knowledge of how to implement a MapsIndoors map with the MapsIndoors JavsScript SDK.
```
Comment (marksdk) [inline collections/_web/v3/guides/show_users_position.md]: It'd be great with a link to a relevant article about the SDK here.

File: collections/_web/v3/guides/show_users_position.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+## How the position is determined
+
+The position is determined by utilizing the [Geolocation API](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API), which all modern browsers expose.
+
+Behind the scenes, the browser determines your position based on a number of factors, including IP address, cell towers, GPS, Wifi access points etc. The implementation varies from browser to browser, and from device to device. There is currently no way to tweak the Geolocation API to use different positioning providers.
+
+All browsers will ask the user for permission to share the location by displaying a prompt. This prompt is a part of the browser, thus not customizable.
+
+Also note that the Geolocation API will only work on https websites (and localhost for development).
```
Comment (marksdk) [inline collections/_web/v3/guides/show_users_position.md]:
```suggestion
Also note that the Geolocation API will only work on `https` websites (and `localhost` for development).
```

File: collections/_web/v3/guides/show_users_position.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+The MapsIndoors JavaScript SDK exposes a `PositionControl` class.
+
+An instantiation of this class will generate a button that, when clicked:
+
+* will start tracking the user's device location
+* show a dot on the map representing location (if accuracy is good enough - more on that later)
+* show a circle representing the position accuracy
+
+Clicking on the button will pan the map, so the current position is in the center of the map.
```
Comment (marksdk) [inline collections/_web/v3/guides/show_users_position.md]:
```suggestion
Clicking on the button will pan the map, so the current position is in the center of the map.

```

File: collections/_web/v3/guides/show_users_position.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+The button will be blue whenever the position is in center of the map.
+
+If user has granted permission indefinetely, the map will pan to the current position when reloading the app *(this does not work in Internet Explorer 11 due to missing support of the [Permissions API](https://developer.mozilla.org/en-US/docs/Web/API/Permissions_API))*.
```
Comment (marksdk) [inline collections/_web/v3/guides/show_users_position.md]:
```suggestion
If the user has granted permission indefinitely, the map will pan to the current position when reloading the app *(this does not work in Internet Explorer 11 due to missing support of the [Permissions API](https://developer.mozilla.org/en-US/docs/Web/API/Permissions_API))*.
```

File: collections/_web/v3/guides/show_users_position.md (hunk 1 above)
Comment (ammapspeople) [reply collections/_web/v3/guides/show_users_position.md]: I added a link.

**Pull request merged**